### PR TITLE
General improvements.

### PR DESCRIPTION
**Pull request by klaasjan-ritense**
This PR is purely meant to improve the existing documentation. I didn't add or remove any pages or anything. Examples:

- removed a number of typos
- added and removed a number of commas
- removed duplicate words
- et cetera

### CODE REVIEW

File: extending-valtimo/plugin/custom-plugin-definition.md [hunk 1]
```diff
@@ -87,27 +87,27 @@ class TwitterPluginFactory(
 ### Plugin categories
 Plugin categories denote a commonality between plugins. They can be applied to interfaces, so any plugin
 implementing this will belong to that category. When a plugin implements more than one interface with a category, it
-belongs to multiple categories. This can be used by backend to create plugins that rely on other plugins to be
-configured, and can be used by frontend to search for plugins in a specific category.
+belongs to multiple categories. This can be used by the backend to create plugins that rely on other plugins to be
```
[the remark this answers is not in the document]
Comment (klaasjan-ritense) [reply extending-valtimo/plugin/custom-plugin-definition.md:90]: As I understand it, 'backend' is treated as a noun here. Hence, 'the'.

File: introduction/modules/form-flow-introduction.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
-# What are form flows
+# Form flow
 
 Form flow allows users to create a sequence of forms, similar to a flow chart or form wizard. A form flow definition
-contains the configuration for each step, what the subsequent steps are, and the conditions under which they can be
+contains the configuration for each step, what the subsequent steps are and the conditions under which they can be
```
Comment (marijnritense) [inline introduction/modules/form-flow-introduction.md:4]: Please leave the Oxford comma in.

Comment (klaasjan-ritense) [reply introduction/modules/form-flow-introduction.md]: Done!

File: introduction/modules/modules.md [hunk 3]
```diff
@@ -1,41 +1,40 @@
...
+Valtimo - like completing a task or uploading a file - are audited by default. It is possible to audit other actions
+by using this feature in the implementation code. The audit trail is available for each case to show the history of
+actions of that case.
 
 ### Connector
 
 Connectors are Valtimo configurable components that can be used to connect to external systems. This module provides a
-framework to create, configure, and use connectors. Certain Valtimo modules include their own connectors that will be
-automatically available for configuration when starting a Valtimo instance.
+framework to create, configure and use connectors. Certain Valtimo modules include their own connectors that become
```
Comment (marijnritense) [inline introduction/modules/modules.md:15]: Same here, please leave the Oxford comma in.

File: extending-valtimo/plugin/custom-plugin-definition.md [hunk 4]
```diff
@@ -87,27 +87,27 @@ class TwitterPluginFactory(
 ### Plugin categories
 Plugin categories denote a commonality between plugins. They can be applied to interfaces, so any plugin
 implementing this will belong to that category. When a plugin implements more than one interface with a category, it
-belongs to multiple categories. This can be used by backend to create plugins that rely on other plugins to be
-configured, and can be used by frontend to search for plugins in a specific category.
```
Comment (marijnritense) [inline extending-valtimo/plugin/custom-plugin-definition.md:91]: Please leave the Oxford comma in.

File: using-valtimo/form-link/configure-task.md [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
-found. What this preliminary step is depends on the type of form that should be linked to a task.
+found. What this preliminary step is, depends on the type of form that should be linked to a task.
 
 ## Configuring a form link through the UI
 
 To create a form link for a start event or user task, the following steps are necessary:
 
-1. In the admin menu, go to "Process links". This shows a  select box, specifying which process and
-   version is being configured, as well as a diagram for the current selection.
+1. In the menu 'Admin', go to "Process links". This shows a select box, specifying which process and
```
Comment (marijnritense) [inline using-valtimo/form-link/configure-task.md:10]: I agree with the quotes here, but I don't know if I agree with switching the order of admin and menu. To me that reads a bit less natural, not sure what others think.

File: using-valtimo/document/start-case.md [hunk 6]
```diff
@@ -19,14 +19,14 @@ definition.
 
 ![Form used to start new case](img/new-case-start-form.png)
 
-After submitting the form a new case is started, and a process is started for the case. The progress of the process can
+After submitting the form, a new case is started and a process is started for the case. The progress of the process can
```
Comment (marijnritense) [inline using-valtimo/document/start-case.md:22]: Can you move back in this Oxford comma as well?

File: using-valtimo/document/link-process-and-case.md [hunk 7]
```diff
@@ -13,8 +13,8 @@ This page requires the following:
 
 To link a process to a case, the following steps are necessary:
 
-1. Go to the Admin menu.
-2. Go to the 'Cases' menu.
+1. Go to the menu 'Admin'.
```
Comment (marijnritense) [inline using-valtimo/document/link-process-and-case.md:16]: I'll keep pointing these out, and we can decide what to do with them later. To me, switching the order of admin and menu feels less natural. The intention is for it to be used as an adjective.

File: using-valtimo/document/custom-case-columns.md [hunk 8]
```diff
@@ -101,7 +101,7 @@ export const environment: ValtimoConfig = {
   ```
 
 For each case type for which custom columns are to be defined, a property is added to the `customDefinitionTables` 
-object. The key of this property is the document definition id, and the value an array of `DefinitionColumn`, i.e.:  
+object. The key of this property is the document definition id and the value an array of `DefinitionColumn`, i.e.:
```
Comment (marijnritense) [inline using-valtimo/document/custom-case-columns.md:104]: Oxford comma

File: using-valtimo/document/create-document-definition.md [hunk 9]
```diff
@@ -65,12 +65,12 @@ To create a document definition, the following steps are necessary:
...
 ### Uploading a document definition
 
-As an alternative for creating a document definition in the project, a document definition can also be uploaded via the
+As an alternative to creating a document definition in the project, a document definition can also be uploaded via the
 admin interface. Creating the file itself is identical, but the path is not important as it will be uploaded to Valtimo.
 
-* Go to the Admin menu.
-* Go to the Cases menu.
-* Click on Upload case-definition.
+* Go to the menu 'Admin'.
```
Comment (marijnritense) [inline using-valtimo/document/create-document-definition.md:71]: Order of words again, Admin and Cases are used as adjectives in this situation.

File: introduction/overview.md [hunk 10]
```diff
@@ -3,22 +3,22 @@
...
+2.	**Service Orchestration**. In large scale microservice landscapes, communication between services can be a challenge. To step away from point-to-point connections, communication can be handled with a choreography pattern with a message bus solution like Kafka, AWS SQS or RabbitMQ. The alternative option is to use an orchestration pattern using a BPMN engine. These types of processes are typically seen in enterprise environments, focussing on eliminating manual work. Typically high volume processes with no or minimal manual work. Using Valtimo for Service Orchestration is not recommended. Using a scalable BPMN engine is advised.
```
Comment (marijnritense) [inline introduction/overview.md:21]: I think `Service Orchestration should` not be capitalized at all in the second to last sentence.

File: introduction/modules/modules.md [hunk 11]
```diff
@@ -215,10 +218,10 @@ a resource in OpenZaak as an *eenvoudig informatie object* (simple information o
 SmartDocuments itself is an external service that deals with generating documents based on templates and placeholders.
 This module offers a plugin which can connect to this service to generate documents.
 
-### Wordpress mail
+### Wordpress email
 
-The wordpress mail plugin is a plugin for the Wordpress CMS that provides a REST API for sending mail. This module 
-includes an implementation of the mail module that uses a connector to send mail through this API.
+The Wordpress email plugin is a plugin for the Wordpress CMS that provides a REST API for sending emails. This module
```
Comment (marijnritense) [inline introduction/modules/modules.md:223]: I think I would rephrase this section to:

```### Wordpress mail

The Wordpress mail plugin is a plugin for the Wordpress CMS that provides a REST API for sending emails. This module
includes an implementation of the 'mail' module, which uses a connector to send emails through this API.```

Comment (marijnritense) [reply introduction/modules/modules.md:223]: The module is called wordpress mail, so I'd like to suggest to stick to that name when talking about the module. Same goes for the mail module.

File: introduction/modules/modules.md [hunk 12]
```diff
@@ -83,21 +86,21 @@ document. This does *not* generate an actual document, but writes the calls to t
...
+example, emails to blacklisted addresses can be filtered out. Dispatching the email after filtering is facilitated by 
+another module.
 
 ### Mandrill
 
-Mailchimp is a SaaS solution for sending email. Mailchimp is the name of the 
+Mailchimp is a SaaS solution for sending emails. Mailchimp is the name of the 
 [transactional mail addon](https://mailchimp.com/features/transactional-email/) that Mailchimp provides. The Valtimo
-Mandrill module provides a mail implementation that uses Mandrill to send mail.
+Mandrill module provides an email implementation that uses Mandrill to send emails.
```
Comment (marijnritense) [inline introduction/modules/modules.md:103]: `Mandrill module provides an mail implementation that uses Mandrill to send emails.`

Since the module is called mail, it should be mail implementation.

File: introduction/modules/modules.md [hunk 13]
```diff
@@ -83,21 +86,21 @@ document. This does *not* generate an actual document, but writes the calls to t
 
 ### Local mail
 
-The local mail module is a stub implementation of the mail module. It is used as a local test implementation that 
-allows the application to start without having to provide an actual implementation. This does *not* send an actual 
-mail.
+The local email module is a stub implementation of the module called 'Mail'. It is used as a local test implementation
```
Comment (marijnritense) [inline introduction/modules/modules.md:89]: `The local mail module is a stub implementation of the 'mail' module. It is used as a local test implementation`

Again, the module is called mail.

File: using-valtimo/form-link/configure-task.md [hunk 14]
```diff
@@ -1,14 +1,14 @@
...
-found. What this preliminary step is depends on the type of form that should be linked to a task.
+found. What this preliminary step is, depends on the type of form that should be linked to a task.
 
 ## Configuring a form link through the UI
 
 To create a form link for a start event or user task, the following steps are necessary:
 
-1. In the admin menu, go to "Process links". This shows a  select box, specifying which process and
-   version is being configured, as well as a diagram for the current selection.
+1. In the Admin menu, go to "Process links". This shows a select box, specifying which process and
```
[the remark this answers is not in the document]
Comment (marijnritense) [reply using-valtimo/form-link/configure-task.md:10]: If we do that, then let's not go for backticks but instead either bold or italicize the text. Backticks right now are reserved for code/file related things. Not sure if using it for this purpose makes it easier to understand or more difficult. Let's not use this PR to figure this out.

**Review by marijnritense — APPROVED** [review]
